pkg/nsx/util/errors.go

```diff
@@ -550,3 +550,11 @@ func CreateConnectionError(host string) *ConnectionError {
 	nsxErr.msg = m
 	return nsxErr
 }
+
+type PageMaxError struct {
```

Comment:
all the errors here before are only used in transport.go for retry. 
you may add comments to mark it 's for client usage.

Reply:
ok